### ISSUE TITLE
Redis Cache Zonal Limitations with other capabilities

### ISSUE THREAD

**Issue by adamyager** [terraform-providers/terraform-provider-azurerm#10863]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) 
RM Version 2.48 and TF version 13.4

### Affected Resource(s)

Redis Cache Premium

### Terraform Configuration Files

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Debug Output

Error: creating Redis Cache "A3DVZZZRDS01T" (Resource Group "A3DVZZZRSG01T"): redis.Client#Create: Failure sending request: StatusCode=0 -- Original Error: Code="BadRequest" Message="Scaling, clustering, and persistence are not yet supported on zone redundant caches\r\nRequestID=01a078a0-ceb6-49b3-be07-a5e9c8f9e1ea"

### References

* #0000

**Comment by catriona-m** [comment]
Thanks for opening this @adamyager. I have tested this config with the latest version of AzureRM Provider and it appears this is now supported so I will close this issue. Please feel free to reopen if you experience this again. Thanks!

**Comment by adamyager** [comment]
I agree @catriona-m  this must have been a defect in the service or the provider.
